### ISSUE TITLE
0^0 should be undefined, not 1

### ISSUE THREAD

**Issue by derivitaandrew** [davidedc/Algebrite#117]
`0^0` is an indeterminate form, so it should throw an error. Instead, it returns `1`.

**Comment by murkle** [comment]
I disagree. There are some references here as to why 1 is a good (pragmatic) choice https://www.geogebra.org/m/rmcxa9un

**Comment by davidedc** [comment]
the tests follow @murkle's view, see power.coffee  :
```
    "(0.0)^(0.0)",
    "1.0",
```
